tools/ctl/ide/explorer/explorer.go

```diff
@@ -157,6 +159,14 @@ func NewExplorer(app *tview.Application, modalFn modalFnType, dex rundex.Reader,
 				}
 			},
 		},
+		{
+			Name: "generate exclusions",
```

Comment:
i think we want a clearer top-line description for this. i didn't immediately understand what this meant and its a bit trickier than the `edit and rebuild` case because there's no "rebuild" action. that means the user needs to understand what they're editing and where it should go which I think we need to work on.